dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml

```diff
@@ -626,6 +626,46 @@
       <artifactId>netty-resolver</artifactId>
       <version>4.2.7.Final</version>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-classes-epoll</artifactId>
+      <version>4.2.7.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-classes-kqueue</artifactId>
+      <version>4.2.7.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-epoll</artifactId>
+      <version>4.2.7.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-epoll</artifactId>
+      <version>jar</version>
+    </dependency>
```

Comment:
Why are we listing these twice? Setting the version tag but also setting it as `jar`? Can this be put to just the version?